irohad/ametsuchi/impl/postgres_block_index.cpp

```diff
@@ -70,10 +72,9 @@ void PostgresBlockIndex::index(const shared_model::interface::Block &block) {
     const auto &creator_id = tx.value().creatorAccountId();
     const TxPosition position{height, static_cast<size_t>(tx.index())};
 
-    makeAccountAssetIndex(creator_id, position, tx.value().commands());
-    indexer_->txHashPosition(tx.value().hash(), position);
+    makeAccountAssetIndex(creator_id, tx.value().hash(), tx.value().createdTime(), position, tx.value().commands());
     indexer_->committedTxHash(tx.value().hash());
-    indexer_->txPositionByCreator(creator_id, position);
+    indexer_->txPositions(creator_id, tx.value().hash(), std::string(), tx.value().createdTime(), position);
```

Comment:
Clear. I will delete the extra data.

Reply:
seems still relevant. what about returning a boolean from `makeAccountAssetIndex` that is `true` if the tx was indexed with some asset and `false` otherwise? then we could index it with `boost::none` as asset only if `makeAccountAssetIndex` returned `false`.